docs/en/tutorials/getting-started.rst

```diff
@@ -616,12 +612,11 @@ domain model to match the requirements:
         }
     }
 
```

Comment:
This section regarding ArrayCollection was confusing-written and likely inaccurate. For instance, no objects of type "Doctrine" are involved here.